src/Transport/Presets/BinaryDownloadPreset.php

```diff
@@ -28,4 +30,19 @@ public static function create(
             BinaryFileDecoder::createWithAutodiscoveredPsrFactories()
         );
     }
+
+    /**
+     * @return TransportInterface<MultiPart, BinaryFile>
+     */
+    public static function fromFormData(
```

Comment:
Maybe this name is not that good given its context and what it operates upon?
What about : `BinaryDownloadPreset::withMultiPartRequest()`?
Any better alternative is welcome :)


I'm also in doubt about keeping `create()` if we could have a more meaningfull `withEmptyRequest`. (we could alias the `create` to that one for now and deprecate its usage?) What do you think?

Reply:
I like your renaming suggestions. I cannot come with better names. I'll apply them.